.changeset/lovely-penguins-rule.md

```diff
@@ -0,0 +1,10 @@
+---
+"hpe-design-tokens": minor
+---
+
+- Removed `color.text.xweak`, `color.icon.xweak`, `color.text.brand`, `color.icon.brand`
+- Removed `checkbox.control.selected.disabled` (use `checkbox.control.disabled.rest`)
+- Removed `radiobutton.control.selected.disabled` (use `radiobutton.control.disabled.rest`)
```

Comment:
Were these the only two affected by moving from `enabled` to `rest`? Any others that we need to call out?